aten/src/ATen/core/ivalue.h

```diff
@@ -386,7 +386,7 @@ struct TORCH_API IValue final {
   // While some of these accessors could be generated through templates,
   // we prefer to write them manually for clarity
 
-  IValue(at::Tensor t) : tag(Tag::Tensor), is_intrusive_ptr(false) {
+  IValue(at::TensorBase t) : tag(Tag::Tensor), is_intrusive_ptr(false) {
```

Comment:
@ngimel this should be a more general solution. It allows `IValue` to be constructed from `TensorBase`.